src/rs_quark_assets/components/LoadingBar.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div class="h-4 w-screen" aria-hidden="true">
+    <div class="relative flex h-2 overflow-hidden text-xs">
+      <label
+        ><span class="sr-only">Loading progress</span>
+        <div v-if="loading > 0" :class="animation" class="absolute h-10"></div>
+      </label>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "DlLoadingBar",
+}
+</script>
+
+<script setup>
+import { ref } from "vue"
+
+const props = defineProps({
```

Comment:
suggestion: remove unused variable
```suggestion
defineProps({
```